### PR TITLE
add scafold for the documentation website

### PR DESCRIPTION
**Pull request by BMartinos**
OHIE-315

### CODE REVIEW

File: docs/.docusaurus/client-modules.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export default [
+  require("infima/dist/css/default/default.css"),
+  require("/home/bradford/Developer/instant/docs/src/css/custom.css"),
```
Comment (BMartinos) [inline docs/.docusaurus/client-modules.js]: This wont work on other developers computers. I also think we can ignore the `.docusaurus` directory. Lets create a gitignore to exclude these from the repo
https://github.com/jembi/openhim-docs/blob/master/.gitignore

Comment (bradsawadye) [reply docs/.docusaurus/client-modules.js]: fixed https://github.com/openhie/instant/pull/9/commits/5a61fd1160b5db992bd221d8a65b58f4ac1eacc6

File: docs/sidebars.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+module.exports = {
+  'docs': {
+    'Introduction': [
+      'introduction/overview',
+      'introduction/community'
+    ],
+    'Architecture': [
+      'architecture/overview'
+    ],
+    'Components': [
```
Comment (rcrichton) [inline docs/sidebars.js]: I think we should rename 'Components' to 'Packages' like it is in the architecture doc. Each package will contain one or more components.

Comment (BMartinos) [reply docs/sidebars.js]: Great thanks Ryan. Think this was some confusion from my side

Comment (bradsawadye) [reply docs/sidebars.js]: fixed in https://github.com/openhie/instant/pull/9/commits/a4d9f6fc9d902d1c89579cdeacd8d70ed79e1347

**Pull request merged**